README.md

```diff
@@ -105,3 +105,28 @@ $ pybabel compile -f -d ./locale/
 If you added a new language for the first time, you need to make sure its target definition (in the `dactyl-config.yml` file) has the MO file in the `locale_file` field.
 
 After that, next time you build the site using Dactyl it should pull the updated translations!
+
+## Issues, Projects, and Project Boards
+
+Use GitHub Issues to report bugs, feature requests, and suggestions. 
+
+If you are a contributor, use GitHub Projects and Project Boards to plan and track updates to xrpl.org. 
+
+### Project Board `xrpl-docs`
+
+The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. The Board only reflects the status of the Issues and contributors must update the status of an issue as it progresses through different stages. 
```

Comment:
I don't think we need to treat "Board" and "Issues" as proper nouns here. But also, I'm not sure what "The board only reflects the status of the issues" is supposed to mean—as opposed to what? It feels like we can just cut that section out and not lose much.

```suggestion
The [`xrpl-docs`](https://github.com/orgs/XRPLF/projects/4) Kanban board is used to plan and track updates to the XRP Ledger Documentation. Contributors must update the status of an issue as it progresses through different stages. 
```